htdocs/core/ajax/flowjs-server.php

```diff
@@ -35,6 +35,9 @@
 if (!defined('NOREQUIRESOC')) {
 	define('NOREQUIRESOC', '1');
 }
+if (!defined('NOCSRFCHECK')) {
```

Comment:
If you disable the CSRF protection, it means a hacker can provide a link that will force your to execute this component. Becasue this is a write component (it create a file on disk), this is dangerous.
We should be able to have it working with NOCSRFCHECK to on.
Did you include the   token=currentToken as parameter by the caller ?